### DISCUSSION TITLE
Delayed availability of sentinel-2-l2a items (resolved)

### DISCUSSION THREAD

**Discussion by TomAugspurger** [microsoft/PlanetaryComputer#369] [Announcements]
Just an FYI that availability of [sentinel-2-l2a](https://planetarycomputer.microsoft.com/api/stac/v1/collections/sentinel-2-l2a) items will be delayed a bit while I update our pipelines to accommodate some updates [ESA started publishing today](https://sentinels.copernicus.eu/en/web/sentinel/-/deployment-of-sentinel-2-processing-baseline-in-version-05.11-on-23-july). I'll post updates here.

**Comment by TomAugspurger** [comment]
Right now it seems like the latest version of sen2cor (which we use to generate the L2A data) doesn't support this new processing baseline. https://forum.step.esa.int/t/sen2cor-supporting-processing-baseline-05-11/42828 is the upstream issue.

**Reply by TomAugspurger** [reply]
I'm going to spin up a pipeline pulling data from another source shortly. This L2A files will be *slightly* different that the previous ones, primarily from the different (better?) choice of DEM and other sen2cor settings. (https://github.com/microsoft/PlanetaryComputer/issues/291, https://github.com/microsoft/PlanetaryComputer/issues/326).

I'll post here once that's running.

**Reply by TomAugspurger** [reply]
I've started up the new pipeline. New items start to show up over the next few hours. Ping me if you spot any anomalies.

These will be just items published on or after 2024-07-24. I'll pick up items from around the transition sometime tomorrow (I need to take some care to not introduce duplicates).

**Reply by TomAugspurger** [reply]
The backfill for 2024-07-23 is in progress.

**Reply by bmcandr** [reply]
>I'm going to spin up a pipeline pulling data from another source shortly. This L2A files will be slightly different that the previous ones, primarily from the different (better?) choice of DEM and other sen2cor settings.

Hey @TomAugspurger, can you expand on this a bit? Do the data before/after 2024-07-23 differ significantly? Just wondering whether we may need to account for this in our pipelines. Thanks!

**Comment by TomAugspurger** [comment]
I wouldn’t say the changes are “significant”. All the values are in the same range, for example.On Sep 27, 2024, at 2:46 PM, Brendan McAndrew ***@***.***> wrote:﻿

I'm going to spin up a pipeline pulling data from another source shortly. This L2A files will be slightly different that the previous ones, primarily from the different (better?) choice of DEM and other sen2cor settings.

Hey @TomAugspurger, can you expand on this a bit? Do the data before/after 2024-07-23 differ significantly? Just wondering whether we may need to account for this in our pipelines. Thanks!
